test/built-ins/Map/prototype/getOrInsertComputed/returns-value-if-key-is-present-different-key-types.js

```diff
@@ -13,7 +13,6 @@ info: |
     a. If p.[[Key]] is not empty and SameValue(p.[[Key]], key) is true, return p.[[Value]].
   ...
 features: [Symbol, arrow-function, upsert]
-flags: [noStrict]
 ---*/
 var map = new Map();
```

Comment:
Using `map.get()` below is confusing. Also, the actual result should be the first argument